uast/uast.go

```diff
@@ -220,6 +220,16 @@ const (
 	AssignmentVariable
 	AssignmentValue
 
+	// AugmentedAssignment is an augmented assignment usually combining the equal operator with
+	// another one (e. g. +=, -=, *=, etc). It is expected that children contains and
+	// AugmentedAssignmentOperator with a child or aditional role for the specific Bitwise or
+	// Arithmetic operator used. The AugmentedAssignmentVariable and AugmentedAssignmentValue roles
+	// have the same meaning than in Assignment
+	AugmentedAssignment
+	AugmentedAssignmentOperator
```

Comment:
Not really, we have the aritmetic "add" operator (ẗhat is only going to be marked as such on aritmetich expressions on most language AST generators) and the assigment (but not an equal operator, at least not in Python or C, it shows as "comparisonEqual" or "Assigment"). I checked Python, C/C++, Java and D ASTs and either they put all those augmented assignments under the same role with an operator subtoken (D, Python) or they add a different node type for every `x=` operator (C++/Java). ASTs should represent semantic concepts more than tokens.